lib/elixir/test/elixir/record_test.exs

```diff
@@ -6,8 +6,6 @@ defmodule RecordTest do
   require Record
   doctest Record
 
-  import ExUnit.CaptureIO
```

Comment:
I removed this because I was getting
```elixir
warning: unused import ExUnit.CaptureIO
  test/elixir/record_test.exs:9
```
when I ran the full test suite